README.md

```diff
@@ -36,9 +36,9 @@ This repository contains a controller that allows you to operate a [Varnish cach
 This controller is not intended to be a replacement of a regular [ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress/). Instead, it is intended to be used between your regular Ingress controller and your application's service.
 
 ```
-┌─────────┐      ┌─────────┐      ┌─────────────┐
+┌─────────┐       ┌─────────┐       ┌─────────────┐
 │ Ingress ├─────▶│ Varnish ├─────▶│ Application │
```

Comment:
Hey! Thanks for the contribution. :)

This looks like a needed change if I view the page in my browser.
But if I open the file in my code editor, this change breaks the previously good rendering.
If I select the ▶ symbol in my browser, it seems to take 2 "spaces". I think this should be one space using a proper monospace font.

Would you mind changing this to a "real" ASCII diagram using `->` instead? I guess that should work around these font rendering issues :D